rest-client/mhs/outbound/Async-Express-Pattern-Message.http

```diff
@@ -1,10 +1,10 @@
 # @name Async_Express_Pattern_Message
 
 POST {{HOSTNAME}}:{{OUTBOUND-PORT}} HTTP/1.1
```

Comment:
here as well I think it should be more like "base url" and not hostname as it also included protocol